keyboards/planck/keymaps/tylerwince/config.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#ifdef AUDIO_ENABLE
+#define STARTUP_SONG SONG(PLANCK_SOUND)
+#endif
+
+#define MIDI_BASIC
+
+#define ENCODER_RESOLUTION 4
+
+/*
+  Set any config.h overrides for your specific keymap here.
+  See config.h options at https://docs.qmk.fm/#/config_options?id=the-configh-file
+*/
+#define TAPPING_FORCE_HOLD
+#define IGNORE_MOD_TAP_INTERRUPT
+
+  #define EECONFIG_RGB_MATRIX (uint32_t *)16
```

Comment:
This is a strange address for the RGB matrix config - it's offset one byte from the keyboard-level EEPROM space, and enters into the user-level EEPROM space by one byte. Is there a reason for this?

It seems the recommendation for separating RGB light and RGB matrix configs is to define this as `(uint32_t) 28` instead: https://docs.qmk.fm/#/feature_rgb_matrix?id=eeprom-storage

@drashna I see you also have this defined in your keymaps. ???

Reply:
Yeah, that should be changed, and it's generated on the EZ configurator.  So it a lot of this code.

Reply:
not sure I understand this -- the whole individual RGB keys here. What was generated is different than the QMK docs so I am not sure what is right for the EZ

Reply:
@drashna would welcome some critique here if there is a better way to do the RGB matrix on the EZ

Reply:
WEll, honestly, the define here isn't really needed, at all.    It would only really be needed, if you wanted to run both rgb matrix and rgblight at the same time.